outline_vpn/outline_vpn.py

```diff
@@ -311,3 +311,64 @@ def delete_data_limit_for_all_keys(self) -> bool:
             f"{self.api_url}/server/access-key-data-limit", verify=False
         )
         return response.status_code == 204
+    def get_all_key_usages(self):
+        """
+        Get usage for all available keys.
+        """
+        try:
+            keys = self.get_keys()
+            usage_data = self.get_transferred_data()
+            key_usage_list = []
+
+            for key in keys:
+                key_id = key.key_id
+                used_bytes = usage_data.get("bytesTransferredByUserId", {}).get(key_id, 0)
+                key_usage_list.append({
+                    "key_id": key_id,
+                    "name": key.name,
+                    "used_bytes": used_bytes
+                })
+
+            return key_usage_list
+        except Exception as e:
+            raise OutlineServerErrorException(f"Failed to get all key usages: {str(e)}")
+
+    def get_key_usage(self, key_name: str):
+        """
+        Get usage for a given key by name.
+        """
+        try:
+            keys = self.get_keys()
+            usage_data = self.get_transferred_data()
+
+            for key in keys:
+                if key.name == key_name:
+                    key_id = key.key_id
+                    used_bytes = usage_data.get("bytesTransferredByUserId", {}).get(key_id, 0)
+                    return {
+                        "key_id": key.key_id,
+                        "name": key.name,
+                        "used_bytes": used_bytes
+                    }
+            raise OutlineServerErrorException(f"Key with name {key_name} not found")
+        except Exception as e:
+            raise OutlineServerErrorException(f"Failed to get key usage for '{key_name}': {str(e)}")
```

Comment:
I'm not sure these 2 are needed since we integrated the usage stat into the get_key/s methods already, so the key should return this information from the beginning.